jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -65,7 +67,10 @@ public class ArrayByteBufferPool implements ByteBufferPool, Dumpable
     private final long _maxHeapMemory;
     private final long _maxDirectMemory;
     private final IntUnaryOperator _bucketIndexFor;
+    private final IntUnaryOperator _bucketCapacity;
     private final AtomicBoolean _evictor = new AtomicBoolean(false);
+    private final ConcurrentMap<Integer, Long> _noBucketDirectSizes = new ConcurrentHashMap<>();
+    private final ConcurrentMap<Integer, Long> _noBucketIndirectSizes = new ConcurrentHashMap<>();
```

Comment:
I'd drop "Sizes" from the name, or replace it with `_noBucketDirectAcquires`, because sizes does not really give what these maps are.

The method is called `recordNoBucketAcquire()`, so the fields should be that too.

Reply:
Renamed fields.